docs/source/reference/volumes.rst

```diff
@@ -0,0 +1,153 @@
+  - Instance volumes: `Local SSDs <https://cloud.google.com/compute/docs/disks/local-ssd>`_ (temporary)
+
+Use cases
+---------
+
+Use volumes for regional storage with high performance. A volume is created in a
+particular region and can only be attached to instances in the same region.
+
+Use :ref:`Cloud Buckets <sky-storage>` for object storage that works across zones, regions, and clouds.
+
+Usage
+-----
+
+Volumes are specified using the :ref:`file_mounts <yaml-spec-file-mounts>` field in a SkyPilot task.
+
+There are three ways to mount volumes:
+
+1. Mount an existing volume
+2. Create and mount a new network volume (reattachable)
+3. Create and mount a new instance volume (temporary)
+
+.. tab-set::
+
+    .. tab-item:: Mount existing volume
+        :sync: existing-volume-tab
+
+        To mount an existing volume:
+
+        1. Ensure the volume exists
+        2. Specify the volume name using ``name: volume-name``
+        3. You must specify the region or zone in the resources section to match the volume's location
+
+        .. code-block:: yaml
+
+          file_mounts:
+            /mnt/path:
+              name: volume-name
+              store: volume
+              persistent: true
+
+          resources:
+            # Must match the volume's location.
+            cloud: gcp
+            region: us-central1
+            # zone: us-central1-a
+
+    .. tab-item:: Create network volume
+        :sync: new-network-volume-tab
+
+        To create and mount a new network volume:
+
+        1. Specify the volume name using ``name: volume-name``
+        2. Specify the desired volume configuration (disk_size, disk_tier, etc.)
+
+        SkyPilot will automatically create and mount the volume to the specified path.
+
+        .. code-block:: yaml
+
+          file_mounts:
+            /mnt/path:
+              name: new-volume
+              store: volume
+              persistent: true  # If false, delete the volume when cluster is downed.
+              config:
+                disk_size: 100  # GiB.
+
+          resources:
+            cloud: gcp
+            region: us-central1
+            # zone: us-central1-a
+
+    .. tab-item:: Create instance volume
+        :sync: new-instance-volume-tab
+
+        To create and mount a new instance volume:
+
+        1. Omit the name field, which will be ignored even if specified
+        2. Specify the desired volume configuration (storage_type, etc.)
+
+        SkyPilot will automatically create and mount the volume to the specified path.
+        Note that instance volumes are temporary and will be lost when the instance is stopped or terminated.
+
+        .. code-block:: yaml
+
+          file_mounts:
+            /mnt/path:
+              store: volume
+              config:
+                storage_type: instance
+                disk_size: 100  # GiB.
+
+          resources:
+            cloud: gcp
+            # region: us-central1
+            # zone: us-central1-a
```

Comment:
These two lines can be removed, as the lifecycle of the instance storage is bound with the VM.